lib/evss/reference_data/intake_sites_response.rb

```diff
@@ -1,25 +0,0 @@
-# frozen_string_literal: true
-
-require 'evss/response'
-
-module EVSS
-  module ReferenceData
-    ##
-    # Model for Intake Sites returned by reference_datea
-    #
-    # @param status [Integer] The HTTP status code
-    # @param response [Hash] The API response
-    #
-    # @!attribute countries
-    #   @return [Array[String]] An array of intake sites
-    #
-    class IntakeSitesResponse < EVSS::Response
-      attribute :separation_locations, Array[Hash]
```

Comment:
one more thing.... do we need to delete this file as well? https://github.com/department-of-veterans-affairs/vets-api/blob/master/app/serializers/evss_separation_location_serializer.rb

Reply:
Yeah, that seems right. I just pushed a commit to do this.